tests/transports/test_all_plugins.py

```diff
@@ -34,7 +34,10 @@
 
 @pytest.fixture(scope='function')
 def tmp_path_remote(tmp_path_factory):
-    """Mock the remote tmp path using tmp_path_factory to create folder start with prefix 'remote'"""
+    """Mock the remote tmp path using tmp_path_factory to create folder start with prefix 'remote'
+    remote prefix was used for clarification,
+    if in future we decided to run these tests against an actual remote machine instead of `localhost`.
+    """
```

Comment:
I think the reason is not that we want to run it in the future on remote. I think it is to have two separate tmp path's, one to imitate local, and one to imitate remote. See for example `test_put_get_empty_string_tree` that uses both fixtures. 

```suggestion
    """Provides a tmp path for mocking local and remote computer directory environment.

    Local and remote directories must be kept separate to ensure proper functionality testing.
    """
```

Reply:
yeah sure. But *also* in case you ever want to run this against a remote server, it's easier this way. 
Since *ideally* one would only need to update that `tmp_path_remote` to point to somewhere on remote.